src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const currentRent = 40;
+  const minDaysOff = 3;
+  const maxDaysOff = 7;
+  const minDiscountRent = 20;
+  const maxDiscountRent = 50;
+
+  if (days >= maxDaysOff) {
+    return currentRent * days - maxDiscountRent;
```

Comment:
The use of constants instead of literals for the discount logic is correctly implemented, which makes the code more maintainable and readable.